internal/quickstart/container.go

```diff
@@ -202,7 +202,9 @@ func (m ContainerModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		m.currentStep += 1
 		shouldSendTrackingEvent = true
 	default:
-		// ignore other messages
+		// delegate other messages
```

Comment:
We need to pass the list messages from the container down into the show SDK model.